evm/chain.py

```diff
@@ -249,6 +249,27 @@ def import_block(self, block):
 
         return imported_block
 
+    def ensure_blocks_are_equal(self, block1, block2):
```

Comment:
The main reason why I kept it as a Chain method is so that we can easily [replace it in tests via the new Chain.configure()](https://github.com/pipermerriam/py-evm/pull/76/files#diff-e29dd936977fd333e7e2c113356591d8R90), but I guess with pytest's monkeypatch fixture we could achieve the same without having to worry about reverting after the test is done. I'm happy to give that a go if you think it's worth turning it into a standalone function